app/models/geocoded_address.rb

```diff
@@ -0,0 +1,10 @@
+class GeocodedAddress < ApplicationRecord
+  belongs_to :order
+
+  geocoded_by :address
+  after_validation :geocode
+
+  def address
```

Comment:
also add a test for this method

Reply:
and maybe move this method to concern because as I see you also add the same method to the order model